05.android/02.installation/docs.en.md

```diff
@@ -47,11 +47,41 @@ If you use Android 4, next you will see the list of permissions required by the
 
 ![](android_install_EN_06.png?cropResize=400,600)
 
-Wait for the installation to complete and press **Done**.
+Wait for the installation procedure to finish. When it is finished, press the **Open** button. If you decide to postpone the launch of the application for another time, press **Done**.
 
-![](android_install_EN_07.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/Installmenu2En.png" />
 
-AdGuard is now installed on your device. You can launch the app by pressing its icon in the list of installed apps.
+You will then be asked to read the **License agreement** and AdGuard's privacy policy. 
+You can participate in product development. 
+To do this, you should check the box *"Allow AdGuard to send automatic error reports"*.
+After you can pre-configure Adguard protection by clicking **Continue**, or do it later by clicking **Skip quick settings**.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installmainEn.png" />
+
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection: 
+- **Comfortable** (Blocks all known Analytics , reduces the time of third-party cookies).
+- **High** (Reduces cookie lifetime, removes tracking parameters from URLS).
+- **Ultimate** (Disables browser features that can be used for your tracking. In some cases, you may experience problems with the operation of the sites).
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/privacylevelEn.png" />
+
+**High** as well as **Ultimate** protection levels available only in **Premium** version.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installpersonaldataEn.png" />
+
+In the next window, you will be given the option to hide irritant elements such as the **Like** and **Social media Widgets**.
```

Comment:
In the next window, you will be given the option to hide irritating elements such as the **Like** and **Social media Widgets**.